pandas/core/indexing.py

```diff
@@ -1888,11 +1888,52 @@ def __setitem__(self, key, value):
 
 
 class _AtIndexer(_ScalarAccessIndexer):
-    """Fast label-based scalar accessor
+    """
+    Access a single value for a row/column label pair.
+
+    Similar to ``loc``, in that both provide label-based lookups. Use
+    ``at`` if you only need to get or set a single value in a DataFrame
+    or Series.
+
+    See Also
+    --------
+    DataFrame.iat
```

Comment:
For the See Also section the format should be `item : description` so move to the description up to the same line and just break where length exceeds 75 characters